iml-system-docker-tests/tests/zfs_test.rs

```diff
@@ -24,10 +15,19 @@ async fn run_test(config: &vagrant::ClusterConfig) -> Result<(), SystemTestError
 
 #[tokio::test]
 async fn test_docker_zfs_setup() -> Result<(), SystemTestError> {
-    let config = vagrant::ClusterConfig::default();
+    let config: Config = Config::default();
+    let config: Config = Config {
+        profile_map: vec![("base_monitored".into(), config.storage_servers())],
+        test_type: TestType::Docker,
+        ntp_server: NtpServer::HostOnly,
+        fs_type: FsType::ZFS,
+        ..config
+    };
```

Comment:
nit

```suggestion
    let config: Config = Config {
        profile_map: vec![("base_monitored".into(), config.storage_servers())],
        test_type: TestType::Docker,
        ntp_server: NtpServer::HostOnly,
        fs_type: FsType::ZFS,
        ..Config::default()
    };
```

Reply:
sure, want me to change it to all of the test files in a follow up patch or would you prefer the change in this patch?